helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "*"
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps
+  verbs:
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - core.giantswarm.io
+  resources:
+    - charts/status
+  verbs:
+    - create
+    - patch
+    - update
+- apiGroups:
+  - core.giantswarm.io
+  resources:
+  - charts
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - core.giantswarm.io
+  resources:
+  - charts/status
+  verbs:
+  - create
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - pods
+  verbs:
+  - "get"
+  - "list"
+- apiGroups:
+  - ""
+  resources:
+  - pods/portforward
```

Comment:
why?

Reply:
I think this is because you came from chart-operator RBAC, right? I think we need to also remove all of the below

Reply:
Oops, should delete that now.

Reply:
Yeah probably copy/paste. `chart-operator` does it fo talk with tiller.

Reply:
Yes this is chart-operator specific because it needs to access tiller via port forwarding.